src/components/ContextualMenu/ContextualMenu.less

```diff
@@ -8,19 +8,20 @@
 
 .ms-ContextualMenu {
   .ms-font-m;
-  .ms-u-normalize;
+  .resetMargins;
+  .resetPadding;
```

Comment:
No worries, would you mind adding in `.ms-u-borderBox` back in and committing it? It makes sense to take out `.resetBoxShadow` so we should be able to merge this in once border-box is set.